### ISSUE TITLE
2021.7 package import fails

### ISSUE THREAD

**Issue by dulitz** [craigjmidwinter/total-connect-client#133]
To replicate:

```
python3 -m venv pyenv
. pyenv/bin/activate
pip install total-connect-client
echo from total_connect_client.client import TotalConnectClient >test.py
python3 test.py
```

Expected behavior: code runs with no error.

Actual behavior:

```
Traceback (most recent call last):
  File "test.py", line 1, in <module>
    from total_connect_client.client import TotalConnectClient
  File "[...]/pyenv/lib/python3.7/site-packages/total_connect_client/client.py", line 8, in <module>
    from location import TotalConnectLocation
ModuleNotFoundError: No module named 'location'
```

Suggested fix:

To work in a package environment, client.py line 8 (and all the similar lines elsewhere) need to be
```
from .location import TotalConnectLocation
```

Have confirmed that the suggested fix does fix the problem. But it breaks tests etc. because they are not currently running in a package environment.

**Comment by dulitz** [comment]
Just to be clear, pip installs packages so that the package directory (in our case `total-connect-client/`) is in PYTHONPATH. The contents of that directory can be imported directly (which makes `import total_connect_client` work).

But `import location` will never work for a standard pip install, because `total-connect-client/total_connect_client` is not in PYTHONPATH and shouldn't be.

**Comment by austinmroczek** [comment]
Thanks for reporting and fixing this issue

**Comment by dulitz** [comment]
You're welcome. Confirmed fixed in 2021.7.1.
